raster/src/main/scala/geotrellis/raster/rasterize/SingleBandRasterRasterizeMethods.scala

```diff
@@ -56,12 +56,12 @@ trait SingleBandRasterRasterizeMethods[T <: Tile, S <: Raster[T]] extends Method
       if (ct.isFloatingPoint) ArrayTile.empty(tile.cellType, re.cols, re.rows)
       else DoubleArrayTile.empty(re.cols, re.rows)
 
-    Rasterizer.foreachCellByGeometry(geom, re, options)(new Callback {
+    geom.foreach(re, options)(new Callback {
       def apply(col : Int, row : Int): Unit = {
         val z = tile.get(col, row)
         mutableTile.setDouble(col, row, fn(z))
       }
     })
-    mutableTile
+    Raster(mutableTile, extent)
```

Comment:
We don't need rasterize methods on `Raster`, this is something people will call on the geometry or feature, so remove this trait.